src/robot/model/keyword.py

```diff
@@ -13,11 +13,17 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+from typing import Sequence, TYPE_CHECKING, Type
 import warnings
 
 from .body import Body, BodyItem
 from .itemlist import ItemList
 
+if TYPE_CHECKING:
+    from robot.model.testcase import TestCase
+    from robot.model.testsuite import TestSuite
+    from robot.model.visitor import SuiteVisitor
```

Comment:
Using `from .testcase import TestCase` would be more consistent with our import style. If there's nothing else the change, I merge this PR and this can be changed later.

Reply:
`TestCase` and `testSuite` will also need to be updated as well to match this import style